lib/core/src/lib/form/services/widget-visibility.service.spec.ts

```diff
@@ -421,6 +421,30 @@ describe('WidgetVisibilityService', () => {
             expect(rightValue).toBe('dropdown_label');
         });
 
+        it('should getRightValue be able to return 0 when rightType is set to value', () => {
```

Comment:
I guess we need a unit test that check how the function behave in case rightValue is null after this change

Reply:
I think we also need to review the same code on left 

  ```
  get leftValue(): any {
        if (this.json.leftValue) {
  ```      
      
I would better check the scenario because it looks inconsistent. We are setting the **rightType** to **value** and then the **value** is null ? Why is it null?